client/app/services/intercom.js

```diff
@@ -23,7 +29,16 @@ export default Ember.Service.extend({
     /* eslint-enable camelcase */
   },
 
+  trackEvent(eventName) {
+    let eventAttribute = Ember.String.camelize(eventName);
+    if (!this.get(eventAttribute))  {
```

Comment:
The idea of how event tracking can help direct bug fixes/feature develop is to run intercom user reports to review the frequency and number of users triggering certain events. I think tracking an event per session make the most sense.